### PR TITLE
Add a database revision makefile recipe

### PR DESCRIPTION
**Pull request by 0livd**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -40,6 +40,10 @@ release: $(DEV_STAMP)
 build-translations:
 	$(VENV)/bin/pybabel compile -d ihatemoney/translations
 
+database-revision:
```
Comment (almet) [inline Makefile]: Should we name this "create-database-revision" ?

Comment (JocelynDelalande) [reply Makefile]: > Should we name this "create-database-revision" ?

+1, othewise LGTM

**Comment by 0livd** [comment]
Changes pushed :)

**Comment by almet** [comment]
Aaaand, we're good to go! Thanks :-)

**Pull request merged**